themes/custom/droopler_theme/templates/block/block--local-actions-block.html.twig

```diff
@@ -1,12 +1,46 @@
-{% extends "@droopler_theme/block/block.html.twig" %}
 {#
 /**
  * @file
- * Theme override for local actions (primary admin actions.)
+ * Default theme implementation to display a block.
+ *
+ * Available variables:
+ * - plugin_id: The ID of the block implementation.
+ * - label: The configured label of the block if visible.
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main content
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ *
+ * @ingroup themeable
  */
 #}
-{% block content %}
-  {% if content %}
-    <nav class="action-links">{{ content }}</nav>
-  {% endif %}
-{% endblock %}
+{%
```

Comment:
I am overriding the entire block, otherwise empty link list will also appear, together with `admin-tabs` region.